### PR TITLE
Completed lc107

### PR DESCRIPTION
**Pull request by kwu25**

### CODE REVIEW

File: kelvinWu/treesBFS/lc107/lc107.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+class TreeNode {
+  constructor(value) {
+    this.value = value;
+    this.left = null;
+    this.right = null;
+  }
+}
+//go through tree level by level
+//add nodes from left to right on each level to a subarray
+//add subarray to beginning of result of array
```
Comment (knrodriguez) [inline kelvinWu/treesBFS/lc107/lc107.js:14]: Clear and concise high-level approach, but missing edge cases.

File: kelvinWu/treesBFS/lc107/lc107.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    for (let i = 0; i < levelSize; i++) {
+      let currentNode = queue.shift();
+      levelArray.push(currentNode.value);
+      if (currentNode.left !== null) queue.push(currentNode.left);
+      if (currentNode.right !== null) queue.push(currentNode.right);
+    }
+    result.unshift(levelArray);
+  }
+  return result;
+};
```
Comment (knrodriguez) [inline kelvinWu/treesBFS/lc107/lc107.js:32]: Don't forget to write it twice to build that muscle memory!

File: kelvinWu/treesBFS/lc107/lc107.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+// [[9 10 5] [7 1] [12]]
+
+var root = new TreeNode(12);
+root.left = new TreeNode(7);
+root.right = new TreeNode(1);
+root.left.left = new TreeNode(9);
+root.right.left = new TreeNode(10);
+root.right.right = new TreeNode(5);
+console.log(`Reverse level order traversal: ${traverse(root)}`);
+
```
Comment (knrodriguez) [inline kelvinWu/treesBFS/lc107/lc107.js:47]: Missing test cases to match edge cases.

**Review by knrodriguez — APPROVED** [review]
